### DISCUSSION TITLE
`cd` builtin on Windows

### DISCUSSION THREAD

**Discussion by tiymat** [lmorg/murex#908] [Ideas]
Recently I've had occasion to use Murex on Windows. I'm not a frequent Windows user, so I was surprised to learn that the Windows equivalent of `pwd` is `cd` without any args. I'm wondering whether it would be worth it to change the `cd` builtin to print the working directory when it's invoked without args. (Possibly only on Windows?)

**Comment by lmorg** [comment]
I'm a little hesitant on that one because it creates functional inconsistencies between the Windows and POSIX builds of Murex which might cause some peoples scripts to fail where they're assuming cross-platform consistency due to sticking with the builtins.

Is there a specific problem you're running into? while `pwd` itself is not a builtin, `$PWD` will behave similarly bar support for any parameters that `pwd` might support. eg
```
» $PWD
/Users/lmorg/dev/go/src/github.com/lmorg/murex
```

**Reply by tiymat** [reply]
Oh, checking `$PWD` directly never occurred to me. That should be fine for my use case. Thanks!
